bibliopixel/drivers/LPD8806.py

```diff
@@ -35,23 +32,23 @@ def __init__(self, num,
         "display": "LPD8806 (SPI Native)",
         "desc": "Interface with LPD8806 strips over a native SPI port (Pi, BeagleBone, etc.)",
         "params": [{
-                "id": "num",
-                "label": "# Pixels",
-                "type": "int",
-                "default": 1,
-                "min": 1,
-                "help": "Total pixels in display."
+            "id": "num",
```

Comment:
we can also do
But then I'm not sure how to manipulierte a already set parameter
```python
from .spi_base_driver import MANIFEST as BASE_MANIFEST

MANIFEST = dict(MANIFEST)
MANIFEST.append(.....)
```

For what is this `MANIFEST` uesd, the `bibliopixel.project` or only documentation?